ghcide/test/data/local-sig-lens/Typeclass.expected.hs

```diff
@@ -0,0 +1,7 @@
+module Typeclass where
+
+f :: a
+f = undefined
+    where
+        g :: Num a => a -> a -> a
+        g a b = a + b
```

Comment:
`Num a` lost here, the same method can show constraints for top-level bindings...

Reply:
Are you saying that the lens cannot deduce `Num a =>` here if the signature is not present? 
I find that ok, constraints belong at the top level, not in local type signatures

Reply:
Sorry for ignoring this...

> Are you saying that the lens cannot deduce Num a => here if the signature is not present?

Yes.

> I find that ok, constraints belong at the top level, not in local type signatures

But it's not correct without constraints.